packages/kbn-failed-test-reporter-cli/failed_tests_reporter/get_failures.ts

```diff
@@ -71,19 +72,35 @@ const isLikelyIrrelevant = (name: string, failure: string) => {
 export function getFailures(report: TestReport) {
   const failures: TestFailure[] = [];
 
+  const commandLine = getCommandLineFromReport(report);
+
   for (const testCase of makeFailedTestCaseIter(report)) {
     const failure = getText(testCase.failure);
     const likelyIrrelevant = isLikelyIrrelevant(testCase.$.name, failure);
 
-    failures.push({
+    const failureObj = {
       // unwrap xml weirdness
       ...testCase.$,
       // Strip ANSI color characters
       failure,
       likelyIrrelevant,
       'system-out': getText(testCase['system-out']),
-    });
+      commandLine,
+    };
+
+    // cleaning up duplicates
+    delete failureObj['command-line'];
+
+    failures.push(failureObj);
   }
 
   return failures;
 }
+
+function getCommandLineFromReport(report: TestReport) {
```

Comment:
Nice hoisting 😉

Reply:
I hope it's not confusing, I make use of these features, and leave the less important stuff to the bottom :)

Reply:
Yeah same here.  You're the only one I've come across, who also uses hoisting like this.  It's my personal preference.  So, imho, GOOD ON YOU! 😉